### ISSUE TITLE
Privacy Badger doesn't forget settings when re-installed

### ISSUE THREAD

**Issue by SwartzCr** [EFForg/privacybadgerfirefox#524]
A user reported that after uninstalling and then re-installing on MacOSX privacy badger remembered the same blocked sites, this isn't the proper action for it, and should be fixed.

**Comment by nyuszika7h** [comment]
I can't say I like that behavior, but in my experience, most addons seem to preserve settings after an uninstall. Couldn't this be solved by #497?

**Comment by SwartzCr** [comment]
no #497 wouldn't solve this. #497 removes all user overrides, the issue here is that between re-installs PB will have the same domains hueristic blocked. That is if you see a third party 3 times and it gets blocked by privacy badger, it will still be blocked if you re-install privacy badger, #497 would only revert it if you had done a user override, but won't reset it to green

**Comment by nyuszika7h** [comment]
Oh, that makes sense.

**Comment by cooperq** [comment]
This is a duplicate of #10
